### PR TITLE
Restore Qedref in runcard and remove alphaem_running

### PR DESCRIPTION
**Pull request by niclaurenti**
Opening this pull request to remove alphaem_running and restoring Qedref.
The new formalism will be, as suggested by @AleCandido:
In the case of fixed alphaem Qedref will be set to nan (or will be absent), while for alphaem running Qedref will be set to a number (and we must check that it is equal to Qref)

### CODE REVIEW

**Review by felixhekhorn — APPROVED** [review]

File: src/banana/data/db.py [hunk 1]
```diff
@@ -56,6 +56,7 @@ class Theory(Base):
     Qmc = Column(Float)
     Qmt = Column(Float)
     Qref = Column(Float)
+    Qedref = Column(Float)
```
Comment (AleCandido) [inline src/banana/data/db.py:59]: The name is a nice pun, but slightly confusing (or not 100% explicit). However, this is even not in our runcard, but completely internal to the benchmarking, so we could even keep it...

**Review by AleCandido — COMMENTED** [review]
Perfectly fine, just one tiny comment

File: src/banana/data/db.py (hunk 1 above)
Comment (niclaurenti) [reply src/banana/data/db.py:59]: Yes I agree but I used the old name that was in banana. I think we can change it once for all. All I can think of is `alphaem_ref` of `Qqedref`. If you have better suggestion I'm all ears

Comment (AleCandido) [reply src/banana/data/db.py:59]: Not really, I'm not so good at naming. 

- `Qqedref` is the more explicit version indeed, that's what I thought in the first place
- `QrefQED` (or every variation like `Qrefqed`) it solves the problem of the two `q`s in a line, and the two scales are alphabetically sorted closer (even if they are already side by side)
- `Qemref` it is a further alternative

Pick your favorite, and if the current one is the one you like the most, feel free to keep it :)

File: src/banana/data/db.py [hunk 2]
```diff
@@ -55,7 +55,8 @@ class Theory(Base):
     Qmb = Column(Float)
     Qmc = Column(Float)
     Qmt = Column(Float)
-    Qref = Column(Float)
+    QrefQED = Column(Float)
+    Qedref = Column(Float)
```
Comment (AleCandido) [inline src/banana/data/db.py:59]: Another tiny remark: you changed the wrong one xD
```suggestion
    Qref = Column(Float)
    QrefQED = Column(Float)
```

**Review by AleCandido — CHANGES_REQUESTED** [review]

**Comment by felixhekhorn** [comment]
@AleCandido shall we drop `develop` here as well?

@niclaurenti I guess you want to make a new release? feel free to do so ... depending on the reply to the above question you can drop `develop` right after

**Pull request merged**